scapy/contrib/gtp_v2.py

```diff
@@ -355,7 +377,7 @@ class ULI_TAI(ULI_Field):
     fields_desc = [
         gtp.TBCDByteField("MCC", "", 2),
         gtp.TBCDByteField("MNC", "", 1),
-        ShortField("TAC", 0),
+        BitField("TAC", 0, 16),
```

Comment:
Why is this change needed?

Reply:
Same comment as https://github.com/secdev/scapy/pull/2488/files#r425166992 
Took the reference from existing field - https://github.com/secdev/scapy/blob/master/scapy/contrib/gtp_v2.py#L362